.github/workflows/execution-plan.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - run: mkdir -p artifacts/
       - name: Plan execution
         run: |
-          ./infrastructure/scripts/execution-plan/plan.sh ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} > artifacts/execution-plan.json
+          ./infrastructure/scripts/execution-plan/plan.sh ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} > artifacts/execution-plan.json 2>artifacts/execution-plan-errors.log
```

Comment:
Why save the errors into a file, instead of outputting them directly? If there was a critical error, the command should exit and then I would look at this build step's output to see what went wrong.

Reply:
updated